src/Files.App/UserControls/StatusBar.xaml

```diff
@@ -393,5 +395,141 @@
 			</Button.Flyout>
 		</Button>
 
+		<StackPanel
+			x:Name="TerminalPanel"
+			Grid.Column="3"
+			VerticalAlignment="Center"
+			x:Load="{x:Bind MainPageViewModel.IsTerminalIntegrationEnabled, Mode=OneWay}"
+			Orientation="Horizontal"
+			Spacing="4">
+
+			<Button
+				x:Name="SyncFolderUp"
+				Height="24"
+				Padding="8,0,8,0"
+				VerticalAlignment="Center"
+				x:Load="{x:Bind MainPageViewModel.IsTerminalViewOpen, Mode=OneWay}"
+				Background="Transparent"
+				BorderBrush="Transparent"
+				Command="{x:Bind MainPageViewModel.TerminalSyncUpCommand}">
```

Comment:
```suggestion
				Command="{x:Bind MainPageViewModel.TerminalSyncUpCommand}"
				ToolTipService.ToolTip="{helpers:ResourceString Name=SyncFolderUp}">
```
Add localized tooltip

Reply:
@XTorLukas there is a good chance these buttons won't end up being included. At this point we're still working on the best way to embed a Terminal into the window.